services/director-v2/src/simcore_service_director_v2/modules/rabbitmq.py

```diff
@@ -73,6 +74,10 @@ def get_rabbitmq_client(app: FastAPI) -> RabbitMQClient:
     return cast(RabbitMQClient, app.state.rabbitmq_client)
 
 
+def get_rabbitmq_client_from_request(request: Request):
```

Comment:
following our fastapi-based best pracices, this should go in 
```services/director-v2/src/simcore_service_director_v2/api/dependencies/rabbitmq.py```
and actually it is already there. so you should use that one

Reply:
ok i will, thanks.